charts/keycloak/templates/route.yaml

```diff
@@ -16,4 +16,8 @@ spec:
     {{ end }}
   to:
     kind: Service
+    {{ if and .Values.maintenancePage.enabled .Values.maintenancePage.on }}
```

Comment:
is the `and` supposed to be between the two values? Or does helm do it differently?

Reply:
`Helm Chart` uses `Golang`'s `template' library;
please take a look at the following resource for your leaning:
- https://pkg.go.dev/text/template#hdr-Functions

Reply:
thanks